Snippets/Rabbit/Rabbit_4/Usage.cs

```diff
@@ -140,4 +141,14 @@ void PrefetchCount(EndpointConfiguration endpointConfiguration)
         #endregion
     }
 
+    void SetClientCertificates(EndpointConfiguration endpointConfiguration)
+    {
+        #region rabbitmq-config-client-certificates
+
+        var transport = endpointConfiguration.UseTransport<RabbitMQTransport>();
+        transport.SetClientCertificates(new X509CertificateCollection());
```

Comment:
if it is assumed knowledge that people will know how to instantiate that collection then that is fine